frontend/context/app.tsx

```diff
@@ -23,6 +24,7 @@ interface ISetAvailableTeamsAction {
 interface ISetConfigAction {
   type: ACTIONS.SET_CONFIG;
   config: IConfig;
+  sandbox_expiry?: string;
```

Comment:
Can we make this camel case as well?

Reply:
Oh wait, I think this should be deleted since it is now in a separate action?
```suggestion
```

Reply:
yep removed